include/gammu-message.h

```diff
@@ -450,6 +450,10 @@ typedef struct {
 	 */
 	gboolean RejectDuplicates;
 	/**
+	 * Whether to skip message.
+	 */
+	gboolean SkipMessage;
+	/**
```

Comment:
I'm not really sure it is good idea to include this in Gammu API as it won't have any use there. This is purely SMSD feature.